### DISCUSSION TITLE
blog/2022/220407-github-pages/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#18] [Announcements]
# blog/2022/220407-github-pages/

Arisa 的个人博客

https://blog.arisa.moe/blog/2022/220407-github-pages/

**Comment by ForsakenDelusion** [comment]
感谢大佬 很有帮助

**Comment by jaywhj** [comment]
配置很详细 👍，另外，给博主推荐个插件，文章底部的显示文章更新日期的组件，可以试试这个 [mkdocs-document-dates](https://github.com/jaywhj/mkdocs-document-dates)，更精准，不依赖 Git。
